.go-env.sh

```diff
@@ -4,12 +4,25 @@ set -Eeuo pipefail
 dir="$(dirname "$BASH_SOURCE")"
 dir="$(readlink -ve "$dir")"
 
-user="$(id -u):$(id -g)"
+src="$dir"
+dst="$dir"
+msys=
+if [ "$(uname -o)" = 'Msys' ];	then
+	msys=1
+	if command -v cygpath > /dev/null; then
+		src="$(cygpath.exe --windows "$dst")"
+	fi
+fi
+windowsContainers=
+if [ "$(docker version --format '{{ .Server.Os }}')" = 'windows' ]; then
+	windowsContainers=1
+	dst='c:\app'
```

Comment:
I think we should probably put a comment here too, something like:

```suggestion
	# normally we'd want this to match $src so error messages, traces, etc are easier to follow, but $src might be on a non-C: drive letter and not be usable in the container as-is 😭
	dst='C:\app'
```

(and drive letters are canonically uppercase, so while I'm here I'll be pedantic too :joy:)